### DISCUSSION TITLE
custom.scss error in Jollyany after update - custom/custom file not found for @import: line: 1,

### DISCUSSION THREAD

**Discussion by vkengo** [templaza/astroid-framework#279] [Q&A] [answered]
Hi, 
I've got an error in Jollyany 3.5.12 and Astroid 2.6.3 on Joomla 3.10.11 after updating, there is a front end error.
 Error: `custom/custom` file not found for @import: line: 1, column: 248: 

I have a custom.scss in (scss/custom) which was working before the update.

What I've tryied:
to rename the file, the site loads - it get's ignored no styles applied.
to have a blank custom.scss, the site loads.
to add one line of style in the new blank file, it comes with the same error nothing else loads.

Any help will be much appreciated.

Thanks.

Viken

**Comment by Chacapamac** [comment, marked as answer]
I @vkengo ,
**If your error is something like that;**
![Screen Shot 2023-01-17 at 4 09 05 PM](https://user-images.githubusercontent.com/1907276/213012931-bdf6730a-a88c-4be7-9e6b-476d8dadec08.jpg)

This 500 error happen if you update to **2.6.3** and leave your **“custom.SCSS”** in the normal **Joomla Root > templates > Your_Template_Name > scss** folder.

You need to **Move** your **“custom.scss”** to the  **Joomla Root > media > templates > site > Your_Template_Name > scss**        folder.

**Beware**, that if you have background images or other links in your **“custom.scss”** using a path like the example under;
_{background-image: url("**../../../**images/construction/overlays-patterns/patterns/dustnscratches.png") }_

**You Need change for**
_{background-image: url(“**../../../../../**images/construction/overlays-patterns/patterns/dustnscratches.png") }_
to reach the images folder as the position/Folder level of the custom.scss change.

You can also trash all Astroid generated css files (e.g. compiled-cb547ba78d812c57cb6d07dba094914a.css)  as they will be generated in the **“media”** folder now....


@sonvnn 
**— It will be great if the update do that automatically.**
 **— I think that shoul go in the documentation as many people use a “custom.scss” file**

**Reply by vkengo** [reply]
Hi @Chacapamac ,

That was the exact issue and solution.

Last place I would think when working with templates, specially as I have been using the Joomla Templates: Customise - Editor. Having the override & custom files in the media folder certainly will need to be documented..

Much appreciated.

**Comment by Chacapamac** [comment]
Yes, I ask @sonvnn if he can put that in the documentation.
Also, it will be great if the system move the custom.scss to  the “media” folder on updtate

Good Day @vkengo !

**Comment by ConfidantCommunications** [comment]
Same problem. Not seeing it in the documentation unfortunately!
